infrastructure/smart-contracts/test/NodeLicenseTinyKeys.mjs

```diff
@@ -429,14 +429,36 @@ export function NodeLicenseTinyKeysTest(deployInfrastructure, poolConfigurations
             await xai.connect(addr1).approve(await nodeLicense.getAddress(), xaiPrice);
 
             // Mint the remaining licenses
-            const tx = await nodeLicense.connect(addr1).mintWithXai(BATCH_SIZE, "", false, ethPrice);
+            const tx = await nodeLicense.connect(addr1).mintWithXai(BATCH_SIZE, "", false, xaiPrice);
             console.log("Gas limit: ", tx.gasLimit.toString());
             const txWait = await tx.wait(1);
             console.log("Gas used: ", txWait.gasUsed.toString());
 
-
             const balanceAfter = await nodeLicense.balanceOf(addr1.address);
             expect(balanceAfter).to.eq(BATCH_SIZE + 1);
         });
+
+        
+        it("Check minting an NFT and receiving it with ETH using the MintTo Function", async function() {
+            const {nodeLicense, addr1, addr2, fundsReceiver} = await loadFixture(deployInfrastructure);
+            const initialBalance = await ethers.provider.getBalance(fundsReceiver.address);
+            const price = await nodeLicense.price(1, "");
+            const totalSupplyBeforeMint = await nodeLicense.totalSupply();
+
+            // Mint an NFT to addr1 from addr2
+            await nodeLicense.connect(addr2).mintTo(addr1.address, 1, "", { value: price });
```

Comment:
Do we actually have coverage for minting with promoCode ?
I feel we should test minting multiple too as well as mint with promoCode, what do you think ?

Reply:
Added two new test cases:

1. Check minting with a promo code using the mintTo function and receiving the correct funds.
2. Check minting an multiple keys and receiving them using the mintTo Function.